insane.go

```diff
@@ -59,7 +59,6 @@ const (
 )
 
 var (
-	StartNodePoolSize      = 128
```

Comment:
Initializing capacity using the StartNodePoolSize variable did not decrease/increase performance, but it does affect the memory consumed, so it was decided to remove this variable. 

I understand that this will break backward compatibility for some applications (applications will have to remove use of the StartNodePoolSize variable).